docs/advanced/mbed_targets.md

```diff
@@ -60,35 +60,35 @@ In this case, `ImaginaryTarget` inherits the properties of both `Target` and `TE
 
 - The value of `ImaginaryTarget.default_toolchain` is `ARM` (from `Target`).
 - The value of `ImaginaryTarget.OUTPUT_EXT` is `hex` (from `TEENSY3_1`).
-- The value of `ImaginaryTarget.core` is `null` (from `Target`, since that's the first parent of `ImaginaryTarget` that defines `core`).
+- The value of `ImaginaryTarget.core` is `null` (from `Target`, because that's the first parent of `ImaginaryTarget` that defines `core`).
 
-Avoid using multiple inheritance for your targets if possible, since it can get pretty tricky to figure out how a property is inherited. If you have to use multiple inheritance, keep in mind that the mbed target description mechanism uses the old (pre 2.3) Python mechanism for finding the method resolution order:
+Avoid using multiple inheritance for your targets if possible because it can get pretty tricky to figure out how a property is inherited. If you have to use multiple inheritance, keep in mind that the mbed target description mechanism uses the old (pre 2.3) Python mechanism for finding the method resolution order:
 
-- Look for the property in the current target.
-- If not found, look for the property in the first target's parent, then in the parent of the parent and so on.
-- If not found, look for the property in the rest of the target's parents, relative to the current inheritance level.
+1. Look for the property in the current target.
+1. If not found, look for the property in the first target's parent, then in the parent of the parent and so on.
+1. If not found, look for the property in the rest of the target's parents, relative to the current inheritance level.
 
 For more details about the Python method resolution order, check [this link](http://makina-corpus.com/blog/metier/2014/python-tutorial-understanding-python-mro-class-search-path).
 
-## core
+## Core
```

Comment:
For all these headers, if they're not capitalized as a property name, don't capitalize them in the title